docs/user-guide/federation/federated-services.md

```diff
@@ -0,0 +1,378 @@
+(e.g. on OpenStack). Simply create all of the clusters that you
+require, in the appropriate cloud providers and/or locations, and
+register each cluster's API endpoint and credentials with your
+Federation API Server (See the
+[federation admin guide](/docs/admin/federation/) for details).
+
+Thereafter, your applications and services can span different clusters
+and cloud providers as described in more detail below.
+
+## Creating a Federated Service
+
+This is done in the usual way, for example:
+
+```
+kubectl --context=federation-cluster create -f services/nginx.yaml
+```
+
+The '--context=federation-cluster' flag tells kubectl to submit the
+request to the Federation API endpoint, with the appropriate
+credentials. If you have not yet configured such a context, visit the
+[federation admin guide](/docs/admin/federation/) or one of the
+[administration tutorials](https://github.com/kelseyhightower/kubernetes-cluster-federation)
+to find out how to do so.
+
+As described above, the Federated Service will automatically create
+and maintain matching Kubernetes services in all of the clusters
+underlying your federation.
+
+You can verify this by checking in each of the underlying clusters, for example:
+
+```
+kubectl --context=gce-asia-east1a get services nginx
+NAME      CLUSTER-IP     EXTERNAL-IP      PORT(S)   AGE
+nginx     10.63.250.98   104.199.136.89   80/TCP    9m
+```
+
+The above assumes that you have a context named 'gce-asia-east1a'
+configured in your client for your cluster in that zone. The name and
+namespace of the underlying services will automatically match those of
+the Federated Service that you created above (and if you happen to
+have had services of the same name and namespace already existing in
+any of those clusters, they will be automatically adopted by the
+Federation and updated to conform with the specification of your
+Federated Service - either way, the end result will be the same).
+
+The status of your Federated Service will automatically reflect the
+real-time status of the underlying Kubernetes services, for example:
+
+```
+kubectl --context=federation-cluster describe services nginx
+```
+
+```
+Name:                   nginx
+Namespace:              default
+Labels:                 run=nginx
+Selector:               run=nginx
+Type:                   LoadBalancer
+IP:         
+LoadBalancer Ingress:   104.197.246.190, 130.211.57.243, 104.196.14.231, 104.199.136.89, ...
+Port:                   http    80/TCP
+Endpoints:              <none>
+Session Affinity:       None
+No events.
+```
+
+Note the 'LoadBalancer Ingress' addresses of your Federated Service
+correspond with the 'LoadBalancer Ingress' addresses of all of the
+underlying Kubernetes services (once these have been allocated - this
+may take a few seconds). For inter-cluster and inter-cloud-provider
+networking between service shards to work correctly, your services
+need to have an externally visible IP address. Service Type:
```

Comment:
I would delete the last sentence; it kind of prescribes a certain use case and makes it seem like there aren't other options.